test/intl/Intl__CollatorTest.mjs

```diff
@@ -24,7 +24,7 @@ var collator = new Intl.Collator(["en-US"], {
 Intl.Collator.supportedLocalesOf([
       "en-US",
       "en-GB"
-    ], undefined);
+    ]);
```

Comment:
There are some unrelated changes in the test output like here - were you using a different/newer ReScript version?